internal/mockstore/mocktikv/pd.go

```diff
@@ -122,6 +122,10 @@ func (c *pdClient) GetLocalTSAsync(ctx context.Context, dcLocation string) pd.TS
 	return c.GetTSAsync(ctx)
 }
 
+func (c *pdClient) GetMinTS(ctx context.Context) (int64, int64, error) {
+	return 0, 0, nil
```

Comment:
Do we need to actually implement it?

Reply:
I think we don't need it for now since there is no test to call it.

Reply:
ok